pkg/api/agentPoolOnlyApi/v20180331/errors.go

```diff
@@ -2,14 +2,14 @@ package v20180331
 
 import "fmt"
 
-// ErrorNilNetworkProfile error
-var ErrorNilNetworkProfile = fmt.Errorf("Network profile can not be nil")
-
 // ErrorNilAgentPoolProfile error
 var ErrorNilAgentPoolProfile = fmt.Errorf("Agent pool profile can not be nil")
 
+// ErrorInvalidNetworkProfile error
+var ErrorInvalidNetworkProfile = fmt.Errorf("ServiceCidr, DNSServiceIP, DockerBridgeCidr should all be empty or neither should be empty")
+
 // ErrorInvalidNetworkPlugin error
-var ErrorInvalidNetworkPlugin = fmt.Errorf("Network plugin should be either \"azure\" or \"kubenet\"")
+var ErrorInvalidNetworkPlugin = fmt.Errorf("Network plugin should be either Azure or Kubenet")
```

Comment:
`Kubenet` or `none`? Also, is it case insensitive?

Reply:
The plugin name is Kubenet, but in the model, we represent it internally as "none". Since the error message is more for customer, I put the name to Kubenet. Case does not matter here.